doc: Bluetooth services docs restructuring (#9019)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/services/hrs_client.rst

   :local:
   :depth: 2

The GATT Heart Rate Service Client is used to retrieve information about heart rate measurements.

Overview
********

The GATT HRS Client can retrieve information such as the sensor location from a device that provides the Heart Rate Service.
It can also configure the Heart Rate Service on a remote device by writing specific values to the Heart Rate Control Point characteristic.

Configuration
*************

> Once a connection with a remote device providing a Heart Rate Service is established, the client needs service discovery to discover Heart Rate Service handles.

Author: How to establish connection?

Reviewer: Application that uses the hrs_client library is responsible for connecting to a peripheral that has the HRS service. Typically it will scan the advertising devices (using the scanning module described in the other file in this PR) and if the scanner detects that an advwertiser supports the HRS service, the application will connect to it and it will start receving the Heart Rate data sutin the hrs_client library.

Author: I would like to add it here as the first paragraph in the 'Configuration' subsection. Please check if I haven't changed the meaning by accident: 
_Any application that uses the GATT HRS Client library should connect to peripherals having the HRS. The application uses the scanning module to detect advertising devices. If the scanner detects that an advertiser supports the HRS, the application will connect to it automatically and will start receiving HRS data._

Reviewer: Yes, I think it looks good :)

If this succeeds, the handles of the Heart Rate Service must be assigned to a HRS client instance using the :c:func:`bt_hrs_client_handles_assign` function.

The application is ready to operate with the remote Heart Rate Service.

Usage

> *****

Author: done :)

> Retrieve data from Heart Rate Service or configure its behavior using the functions listed below:

Reviewer 2: ```suggestion
Retrieve data from Heart Rate Service or configure its behavior using the following functions:
```

> Retrieve data from a Heart Rate service or configure its behavior using the following operations:

Reviewer 3: ```suggestion
Retrieve data from the Heart Rate service or configure its behavior using the following operations:
```

> * :c:func:`bt_hrs_client_measurement_subscribe` - Enable notifications for the Heart Rate Measurement characteristic.
>   The notifications are passed to the provided callback function.

Reviewer 2: ```suggestion
* :c:func:`bt_hrs_client_measurement_subscribe` - Enable notifications for the Heart Rate Measurement characteristic.

  The notifications are passed to the provided callback function.
```

  You will receive the current heart rate measurement data.

> * :c:func:`bt_hrs_client_sensor_location_read` - Read a heart rate sensor location.
>   Check the possible locations in :c:enum:`bt_hrs_client_sensor_location`.

Reviewer 2: ```suggestion
* :c:func:`bt_hrs_client_sensor_location_read` - Read a heart rate sensor location.

  Check the possible locations in :c:enum:`bt_hrs_client_sensor_location`.
```

> * :c:func:`bt_hrs_client_control_point_write` - Configure a remote Heart Rate Service.
>   Check the possible values for a Heart Rate Control Point characteristic in :c:enum:`bt_hrs_client_cp_value`.

Reviewer 2: ```suggestion
* :c:func:`bt_hrs_client_control_point_write` - Configure a remote Heart Rate Service.

  Check the possible values for a Heart Rate Control Point characteristic in :c:enum:`bt_hrs_client_cp_value`.
```

* :ref:`central_and_peripheral_hrs`
* :ref:`bluetooth_central_hr_coded`

Additional information
**********************

.. note::
   Do not use any of the values in the :c:struct:`bt_hrs_client` object structure directly.
   All values that should be accessed have accessory functions.
   The structure is fully defined because the application must be able to allocate memory for it.

Dependencies
************

> 

Reviewer 4: Line 63 and 64:
 * :file:\`include/bluetooth/gatt.h\`
 * :file:\`include/bluetooth/conn.h\`
